src/Tizen.Network.Stc/Tizen.Network.Stc/StcManager.cs

```diff
@@ -0,0 +1,112 @@
+    public sealed class SafeStcHandle : SafeHandle
+    {
+        internal SafeStcHandle() : base(IntPtr.Zero, true)
+        {
+        }
+
+        /// <summary>
+        /// Checks the validity of the handle.
+        /// </summary>
+        /// <value>Represents the validity of the handle.</value>
+        public override bool IsInvalid
+        {
+            get
+            {
+                return this.handle == IntPtr.Zero;
+            }
+        }
+
+        /// <summary>
+        /// Release the Stc handle
+        /// </summary>
+        /// <returns>True if the handle is released successfully, otherwise false.</returns>
+        protected override bool ReleaseHandle()
+        {
+            Interop.Stc.Deinitialize(this.handle);
+            this.SetHandle(IntPtr.Zero);
+            return true;
+        }
+    }
+
+    /// <summary>
+    /// A class which is used to manage Smart Traffic control (Stc).<br/>
+    /// </summary>
+    public static class StcManager
+    {
+        /// <summary>
+        /// Gets the Stc safe handle.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <returns>The instance of the SafeStcHandle.</returns>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown when the Stc is not supported.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when the permission is denied.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the method failed due to an invalid operation.</exception>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        internal static SafeStcHandle GetStcHandle()
+        {
+            return StcManagerImpl.Instance.GetSafeHandle();
+        }
+
+        /// <summary>
+        /// Gets the statistics information of an application, which matched the rule, asynchronously. Application ID must be provided.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <param name="appId"> Application's appId for which statistics information is required.</param>
+        /// <param name="rule"> The StcRule object.</param>
+        /// <returns>The Statistics information of the application that used network in between timestamps specified.</returns>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown when the Stc is not supported.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when the permission is denied.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the method failed due to an invalid operation.</exception>
+        /// <exception cref="ArgumentException">Thrown when the method is provided with invalid argument.</exception>
+        public static Task<NetworkStatistics> GetStatisticsAsync(string appId, StcRule rule)
```

Comment:
I have a question, 
How does TimePeriod of StcRule affect the return value?
Is it valid for this API?

Reply:
TimeInterval(From and To) indicates the Range of time for which we require statistical data. Whereas, 'TimePeriod' is to granulate that data in further pieces. That is the statistics data will be chunked in periods of Month or Week or Hour.

Reply:
According to your comment, The statistical data are split to some pieces.
So, the return value should be represented as collection of Network Statistics
Isn't it?